Users/Tutorial.rst

```diff
@@ -351,7 +351,7 @@ To get help on using a bear or to get a description of the bear, use the
     coala --bears=SpaceConsistencyBear --show-bears --show-description
```

Comment:
i added an intentional space between ``/`` and the number, you should remove it and use that link